docs/tutorials/cargo.md

```diff
@@ -0,0 +1,117 @@
+# Setting up your Cargo.toml file for ICU4X
+
+ICU4X makes heavy use of small crates and Cargo features in order to be highly modular. This tutorial is intended to help you set up a Cargo.toml file to download what you need for ICU4X.
+
+## Basic Cargo.toml with testdata
+
+The most basic Cargo.toml to get you off the ground is the following:
+
+```toml
+[package]
+name = "demo_testdata"
+version = "0.1.0"
+edition = "2021"
+publish = false
```

Comment:
I don't think these keys are very relevant to the example, given that you also link to the actual `Cargo.toml`.

Reply:
Do you suggest I remove the entire `[package]` section, or only the `publish =`?